e2e/__tests__/jest.config.js.test.ts

```diff
@@ -10,7 +10,7 @@ import {wrap} from 'jest-snapshot-serializer-raw';
 import runJest from '../runJest';
 import {cleanup, extractSummary, writeFiles} from '../Utils';
 
-const DIR = path.resolve(__dirname, '../jest.config.js');
+const DIR = path.resolve(__dirname, '../tmp/jest-config-js');
```

Comment:
I meant
```
import {tmpdir} from 'os';`
```

😅

Reply:
@SimenB If I move it to `tmp` the `traverses directory tree up until it finds jest.config` test will fail because it can't find the `slash` package.

Reply:
we should install `slash` in it, then. We have a `runYarn` helper already - I don't think it takes args, but that should be a simple refactor.

We can do it later, though 🙂

Reply:
Ah nice, `runYarn` is exactly what I was looking for

Reply:
@SimenB I've reverted the change with the `tmp` bacause I couldn't make `runYarn` to install the `slash` package. I even tried `run('yarn install`)` with same `module does not exist` error.

As you said, we can do it later so we don't need to delay this fix.